[FAB-18188] Log orderer and peer cert expiration date upon startup

denyeart opened the pull request:
#### Type of change

- Improvement (improvement to logging)

#### Description

FAB-17000 added a warning if expiration is within a week.

This change additionally info logs the future enrollment, tls server, and tls client
cert expiration dates upon startup, regardless of whether they will expire within a week or not.

Signed-off-by: David Enyeart <enyeart@us.ibm.com>

JonathanLevi commented:
Quick status: I can't read the (full) CI logs.

> The log length has exceeded the limit of 4 MB (this usually means that the test suite is raising the same exception over and over).
> 
> The job has been terminated

adecaro commented:
@JonathanLevi I'm surprised that test scenario was active. I had explicitly removed it in one of my latest PR. I would recommend to remove *client_tcert_pool_mt.go*, that kind of optimization should be moved, at this point, to the client-sdk, no?

JonathanLevi commented:
Thank you @adecaro. Whether the MT tests are moved to the client-sdk or otherwise at a later point, yes, I, too, am happy to disable these crypto client MT tests altogether at this point. More explicitly:

@christo4ferris, @binhn, @srderson, et al., in view of the above, unless there's any objection from your end - I too believe that we should not enforce the above removed/disabled (MT) scenarios at this point.

christo4ferris commented:
LGTM @ghaskins or @binhn would you please merge this? Given that it is the only PR that is building, I think we can say that this addresses the issue with the build as advertised. Thanks @JonathanLevi for tracking this down.

ghaskins commented:
LGTM

Reviewed-by: Greg Haskins <gregory.haskins@gmail.com>

JonathanLevi commented:
(Good morning and) thanks guys! (@rameshthoomu included! ;-))

denyeart committed:
[FAB-18188] Log orderer and peer cert expiration date upon startup (#1804)

FAB-17000 added a warning if expiration is within a week.

This change additionally info logs the future enrollment, tls server, and tls client
cert expiration dates upon startup, regardless of whether they will expire within a week or not.

Signed-off-by: David Enyeart <enyeart@us.ibm.com>
